src/components/hooks/useMeeting.js

```diff
@@ -33,9 +36,10 @@ export default function useMeeting(meetingID, meetingDestination) {
         .pipe(
           flatMap(({ID}) => meetingsAdapter.getMeeting(ID)),
           take(1),
-          delayWhen(({ID}) => from(meetingsAdapter.addLocalMedia(ID)))
+          delayWhen(({ID}) => from(meetingsAdapter.addLocalMedia(ID))),
+          flatMap(({ID}) => meetingsAdapter.getMeeting(ID))
```

Comment:
Why did you move this after adding local media? I thought the idea of needing this here was because the event subscription needs to happen before the event (adding local media)

Reply:
so I couldn't come up with a better approach than resubscribing to the meeting after addLocalMedia. basically what I need to do is to trigger `addLocalMedia` only once after `getMeeting` subscription.

Reply:
i didn't move it. I appended another `getMeeting`

Reply:
We can address this outside of the PR, but I something tells me this is not the right thing to do